bitcoin/src/p2p/mod.rs

```diff
@@ -91,6 +91,10 @@ impl ServiceFlags {
     /// See BIP159 for details on how this is implemented.
     pub const NETWORK_LIMITED: ServiceFlags = ServiceFlags(1 << 10);
 
+    /// P2P_V2 indicates that the node supports the P2P v2 encrypted transport protocol.
+    /// See BIP324 for details on how this is implemented.
+    pub const P2P_V2: ServiceFlags = ServiceFlags(1 << 11);
```

Comment:
Checked against https://github.com/bitcoin/bips/blob/master/bip-0324.mediawiki#signaling-v2-support

Reply:
Thank you, but nobody will ever see this because I am required to click "resolve" before merging.

Reply:
I meant it as a quick link for other reviewers to use. It also occurred to me we could click unresolve after merging but that's a bit weird and annoying.